examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb

```diff
@@ -1042,88 +1042,88 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from merlin.models.tf.core.transformations import LogitsTemperatureScaler
</pre><p>Should LogitsTemperatureScaler be available from merlin.models.tf ? It looks like other transformations are exposed there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>